drivers/clock_control/clock_stm32_ll_u5.c

```diff
@@ -364,6 +377,10 @@ void config_src_sysclk_pll(LL_UTILS_ClkInitTypeDef s_ClkInitStruct)
 	LL_RCC_HSE_Disable();
 
 #elif STM32_PLL_SRC_HSI
+	set_regu_voltage(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC);
+	while (LL_PWR_IsActiveFlag_VOS() == 0) {
```

Comment:
Check is done in the function. can be removed.

Reply:
Sure, thanks!

Reply:
Fixed